### PR TITLE
Add `2FAuth` to `community` train

### PR DESCRIPTION
**Pull request by stavros-k**
Closes https://github.com/truenas/charts/issues/1277

### CODE REVIEW

File: library/ix-dev/community/two-f-auth/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: two-f-auth
```
Comment (Qubad786) [inline library/ix-dev/community/two-f-auth/Chart.yaml:1]: Should we name it something different?

Comment (stavros-k) [reply library/ix-dev/community/two-f-auth/Chart.yaml:1]: Would two-fauth be okay ? or just one word twofauth?
We can't start with a digit

Comment (Qubad786) [reply library/ix-dev/community/two-f-auth/Chart.yaml:1]: `twofactor-auth` perhaps?

Comment (stavros-k) [reply library/ix-dev/community/two-f-auth/Chart.yaml:1]: I'm fine with that, yea

**Review by Qubad786 — APPROVED** [review]

**Pull request merged**